silkworm/silkrpc/core/receipts.cpp

```diff
@@ -22,16 +22,7 @@
 namespace silkworm::rpc::core {
 
 Task<Receipts> get_receipts(const core::rawdb::DatabaseReader& db_reader, const silkworm::BlockWithHash& block_with_hash) {
-    const auto cached_receipts = co_await core::rawdb::read_receipts(db_reader, block_with_hash);
-    if (!cached_receipts.empty()) {
-        co_return cached_receipts;
-    }
-
-    // If not already present, retrieve receipts by executing transactions
-    // auto block = co_await core::rawdb::read_block(db_reader, hash, number);
-    // TODO(canepat): implement
-    SILK_WARN << "retrieve receipts by executing transactions NOT YET IMPLEMENTED";
-    co_return Receipts{};
+    co_return co_await core::rawdb::read_receipts(db_reader, block_with_hash);
```

Comment:
I suggest to restore something similar to the previous code here with a single difference being the condition to check is on `cached_receipts` equal to `std::null_opt` or not. Then, you should re-add the warning because we do not currently have implemented transaction re-execution here, but at least empty receipts are returned with such warning when we should recompute them instead.